tests/plugins/splitbutton/splitbutton.js

```diff
@@ -3,46 +3,108 @@
 ( function() {
 	'use strict';
 
-	bender.editor = {
-		config: {
-			on: {
-				pluginsLoaded: function( evt ) {
-					var editor = evt.editor,
-						counter = {
-							faceClick: 0,
-							itemClick: 0
-						},
-						items = [ {
-							command: 'bold',
-							icon: 'bold',
-							'default': true
-						}, {
-							command: 'italic',
-							icon: 'italic'
-						}, {
-							command: 'underline',
-							icon: 'underline'
-						}, {
-							command: 'strike',
-							icon: 'strike'
-						}, {
-							command: 'subscript',
-							icon: 'subscript'
-						} ];
-
-					editor.ui.add( 'teststylesplit', CKEDITOR.UI_SPLITBUTTON, {
+	bender.editors = {
+		splitButtonAsPlugin: {
+			config: {
+				on: {
+					pluginsLoaded: function( evt ) {
+						var editor = evt.editor,
+							counter = {
+								faceClick: 0,
+								itemClick: 0
+							},
+							items = [ {
+								command: 'bold',
+								icon: 'bold',
+								'default': true
+							}, {
+								command: 'italic',
+								icon: 'italic'
+							}, {
+								command: 'underline',
+								icon: 'underline'
+							}, {
+								command: 'strike',
+								icon: 'strike'
+							}, {
+								command: 'subscript',
+								icon: 'subscript'
+							} ];
+
+						editor.ui.add( 'teststylesplit', CKEDITOR.UI_SPLITBUTTON, {
+							items: items
+						} );
+
+						editor.ui.add( 'staticface', CKEDITOR.UI_SPLITBUTTON, {
+							face: {
+								command: 'bold',
+								icon: 'bold'
+							},
+							items: items
+						} );
+
+						editor.ui.add( 'customclick', CKEDITOR.UI_SPLITBUTTON, {
+							face: {
+								icon: 'underline',
+								click: function() {
+									counter.faceClick++;
```

Comment:
The whole `counter` concept could be simplified by using sinon's stubs. Since you're using them in tests, you could just define stubs in test-scope or in context of `tests` object.

Then you'd want to reset it in `setup/teardown` function to make sure that test can not be affected by other unit tests.